.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -10,9 +10,9 @@ on:
 env:
   PO4A_VERSION: "0.71"
   PO4A_GH_URL: https://github.com/mquinson/po4a/releases/download
-  REPO_URL: https://github.com/Heegu-sama/Homm3BG
   GHOSTSCRIPT_URL: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10030/gs_10.03.0_amd64_snap.tgz
   ARTIFACT_RETENTION_DAYS: 1
+  HOMM3_GITHUB_BUILD: 1
```

Comment:
Didn't work. Please try putting it in the `pre_complie:` arg in every step which builds the PDF file.

Reply:
Yeah, I noticed. I kind of expected this would fail but still wanted to give it a try and avoid unnecessary duplication

Reply:
Confirmed 👍🏻